src/wallet/rpc/wallet.cpp

```diff
@@ -701,6 +701,159 @@ RPCHelpMan simulaterawtransaction()
     };
 }
 
+static RPCHelpMan setdescriptorhdkey()
+{
+    return RPCHelpMan{"setdescriptorhdkey",
```

Comment:
Any reason not to overload `sethdseed` for this?

Reply:
Same question, couldn't `sethdseed` just have an option where it doesn't add fresh descriptors?

Reply:
Changed to overload `sethdseed`.

Reply:
Don't forget to update the PR description and the relevant error message in `createwalletdescriptor`.